app/controllers/spree/api/v2/storefront/passwords_controller.rb

```diff
@@ -3,6 +3,8 @@ module Api
     module V2
       module Storefront
         class PasswordsController < ::Devise::PasswordsController
+          skip_before_action :verify_authenticity_token
+
```

Comment:
```suggestion
        class PasswordsController < ::Spree::Api::V2::BaseController
```

we need to inherit after API v2 base controller